power.md

```diff
@@ -55,19 +55,20 @@ When executing the script with the '-h' command line parameter, the user is
 presented with help text:
 
 ```bash
-# power.py  -h
+# power.py -h
 
-/usr/local/sbin/power.py -i -M <maxfreq> -m <minfreq> -s <setfreq> -r <range> -g <governor>
+power.py -i -M <maxfreq> -m <minfreq> -s <setfreq> -r <range> -g <governor>
    <no params>   use interactive menu
    -h            Show this help
    -i            Show information on available freqs, C-States, etc
+   -f            Show UNCORE fabric information
    -l            List information on each core
-   -L <sec>      List information on each core repeatedly at <sec> intervals
+   -L <sec>      List information repeatedly at <sec> intervals (need -l or -f, or both)
    -M <freq>     Set core maximum frequency. Can also use "max", "min", or "base"
    -m <freq>     Set core minimum frequency. Can also use "max", "min", or "base"
    -s <freq>     Set core frequency (within min and max)
    -r <range>    Range of cores to affect, e.g. 1-3,5,7
-   -g <governor> Set core governor (usually 'userspace')
+   -g <governor> Set P-State governor (usually 'userspace')
```

Comment:
Any reason for removing "core", could it be more informative for "Set per core P-state "